test/TMxpEntityTagHandlerTest.cpp

```diff
@@ -0,0 +1,162 @@
+/***************************************************************************
+ *   Copyright (C) 2022 by Gustavo Sousa - gustavocms@gmail.com            *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+#include <QTest>
+#include "TMxpEntityTagHandler.h"
+#include "TMxpStubClient.h"
+#include <TMxpTagParser.h>
+#include <TMxpTagProcessor.h>
+#include <TMxpProcessor.h>
+
+class TMxpEntityTagHandlerTest : public QObject {
+Q_OBJECT
+
+private:
+
+private slots:
```

Comment:
Is the first `private` line 30 redundant though, or does the compiler ignore the private on line 32 because there is something else between it and the `:`? Of course `private`/`protected`/`public` as applied to slot methods is only relevant ***when the slot is called directly as a function/method*** they are always `public` when called through Qt's signal/slot infrastructure...